core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1865,40 +1869,41 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
       }
     }
 
+    traceDBIOWithParent("requireCreateWorkspaceAccess", parentSpan)(s0 => requireCreateWorkspaceAccess(workspaceRequest, dataAccess, s0) {
```

Comment:
github doesn't show the diff for this block very well. it's basically just the addition of:
`      traceDBIOWithParent("maybe requireBillingProjectOwnerAccess", s0) (s1 => requireBillingProjectOwnerAccess(workspaceRequest) {`
and auto-indenting

<img width="2160" alt="Screen Shot 2020-10-05 at 5 39 50 PM" src="https://user-images.githubusercontent.com/5375000/95135146-02b35b00-0732-11eb-8e25-e9995988900b.png">